crates/bevy_audio/src/lib.rs

```diff
@@ -119,3 +120,47 @@ impl AddAudioSource for App {
         self
     }
 }
+
+/// Command for playing a standard bevy audio asset
+pub struct AudioSpawnCommand<'a> {
+    /// Path to the sound asset
+    pub path: AssetPath<'a>,
+    /// Sound playback settings
+    pub settings: PlaybackSettings,
+}
+
+impl Command for AudioSpawnCommand<'static> {
+    fn apply(self, world: &mut World) {
+        // do whatever you want with `world` and `self.data` here
+        let asset = world.get_resource::<AssetServer>().unwrap();
+        let source = asset.load(&self.path);
+        world.spawn(AudioBundle {
+            source,
+            settings: self.settings,
+        });
+    }
+}
+
+/// Trait for playing sounds with commands
+pub trait AudioSpawnCommandExt {
+    /// Command for playing a standard bevy audio asset with default settings
```

Comment:
There should be a comment here that you need to preload the asset if you want the sound to play without delay.

Reply:
Done